adaptive_breakpoints/lib/src/adaptive_breakpoints.dart

```diff
@@ -90,6 +100,23 @@ extension AdaptiveWindowTypeExtension on AdaptiveWindowType {
         throw AssertionError('Unsupported AdaptiveWindowType');
     }
   }
+
+
+  bool operator <=(Object other) {
+    return _operatorMap[this] <= _operatorMap[other];
+  }
+
+  bool operator <(Object other) {
+    return _operatorMap[this] < _operatorMap[other];
+  }
+
+  bool operator >=(Object other) {
+    return _operatorMap[this] >= _operatorMap[other];
+  }
+
+  bool operator >(Object other) {
+    return _operatorMap[this] > _operatorMap[other];
+  }
```

Comment:
You might  even be able to use the `=>` syntax to make it even shorter? Not sure if this will work but worth a try
```suggestion
  bool operator >(Object other) => _operatorMap[this] > _operatorMap[other];
```

Reply:
Done